index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <style>
+  @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap');
+      </style>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
+</head>
+<body>
+    <section class="navigation">
+        <nav class="header">
+            <p class="logo">Josiah</p>
+            <a href="#" class="menu"><img src="img/menu.png" alt=""></a>
+        </nav>
+    </section>
+
+    <section class="bio-section">
+        <div class="bio">
+            <h3 class="bio-header">I'm Josiah <br>Glad to see you</h3>
```

Comment:
Well done in using an h tag here but I suggest you change this to an h1 tag instead of an h3 tag. This will improve SEO scores for this page and is more semantic. Check [this link](https://www.reliablesoft.net/h1-tag/) for more information about this 👍